test/GroupByTest.cpp

```diff
@@ -29,10 +29,14 @@
 #include "util/IndexTestHelpers.h"
 
 using namespace ad_utility::testing;
+using ::testing::Eq;
+using ::testing::Optional;
 
 namespace {
 auto I = IntId;
-}
+// Wrapper type so that we can safely use `IdTable` within gtest matchers.
```

Comment:
We already have this shortcut for `I(12)` instead of `Id::makeFromInt(12)`,
So you could consistently use it.